src/main/resources/assets/wurst/lang/zh_cn.json

```diff
@@ -1,139 +1,139 @@
-  "description.wurst.hack.feedaura": "自动喂食你附近的动物.",
-  "description.wurst.hack.fightbot": "仿照机器人会自动跟随并击杀最近的实体.\n很适合刷怪区域.",
-  "description.wurst.hack.fish": "关闭水底的重力,所以你可以像鱼一样游泳.",
-  "description.wurst.hack.flight": "允许你飞天.\n\n§c§l警告:§r 如果你不使用 NoFall ,会导致受到掉落伤害.",
-  "description.wurst.hack.follow": "利用机器人方式会接近一个实体.\n非常惹恼人.\n\n使用 .follow 指令跟随特定的实体.",
-  "description.wurst.hack.forceop": "破解 AuthMe登录插件 密码.\n一般用于获得OP.（PS：只是撞库跑密码的软件）",
-  "description.wurst.hack.freecam": "允许在不移动你角色的情况下自由移动..",
-  "description.wurst.hack.fullbright": "允许你在黑暗中看的一清二楚.",
-  "description.wurst.hack.glide": "使你在下落时缓慢下降.\n\n§c§l警告:§r 如果你不使用 NoFall 则可能会摔落致死..",
-  "description.wurst.hack.handnoclip": "允许你能透过墙壁触碰到特殊的方块.",
-  "description.wurst.hack.headroll": "让你长期点头.\n只有其他玩家能看到.",
-  "description.wurst.hack.healthtags": "在玩家名字旁边显示血量.",
-  "description.wurst.hack.highjump": "允许你跳的更高.\n\n§c§l警告:§r 如果你不使用 NoFall 则可能会摔落致死.",
-  "description.wurst.hack.infinichat": "移除聊天栏的 256 字符限制.\n一般用于编辑NBT的指令.\n\n§6§l注意:§r 不推荐用于和别人聊天.大多数服务器一般超过256字符就会被砍掉.",
-  "description.wurst.hack.instantbunker": "建造一个小小的火柴盒. 需要57个方块.",
-  "description.wurst.hack.itemesp": "高亮附近的掉落物品.",
-  "description.wurst.hack.itemgenerator": "生成随机的物品并将他们丢在地上.\n§o仅限创造模式.§r",
-  "description.wurst.hack.jesus": "允许你走在水面上.\n耶稣曾经用这个功能 约~2000 年前.",
-  "description.wurst.hack.jetpack": "模仿你好像真的有飞行背包一样飞行.\n\n§c§l警告:§r 如果你不使用 NoFall ,会导致受到掉落伤害..",
-  "description.wurst.hack.kaboom": "像爆炸一样破坏你附近的方块.\n这个可能比 Nuker 功能破坏更快如果服务器没有反作弊的话. 如果使用适合的工具和破坏脆弱的方块的效果会更好.\n注意: 这并不是真正的爆炸.",
-  "description.wurst.hack.killauralegit": "KillauraLegit 使得更难检测.\n在正常的NoCheat+(NCP)服务器上不需要!",
-  "description.wurst.hack.killaura": "自动攻击你附近的实体.",
-  "description.wurst.hack.killpotion": "生成一个能杀死绝大多数生物的药水,包括创造模式的玩家.但对那些不死的生物或者已死生物无效.\n\n需要在创造模式下.",
-  "description.wurst.hack.liquids": "允许你在水中直接放方块.",
-  "description.wurst.hack.lsd": "产生吃了云南的蘑菇一样的效果.",
-  "description.wurst.hack.masstpa": "给每个人发送tpa请求.\n如果有人接了就停下来.",
-  "description.wurst.hack.mileycyrus": "就是类似疯狂按Shift潜行.",
+  "description.wurst.hack.anchoraura": "自动放置（可选）并引爆重生锚，杀死你附近的生物。",
+  "description.wurst.hack.antiafk": "随机走动，用于避免服务器挂机检测。\n需要至少有 3x3 的区域可以走动。",
+  "description.wurst.hack.antiblind": "防止失明。不兼容 OptiFine。",
+  "description.wurst.hack.anticactus": "保护你免受仙人掌的伤害。",
+  "description.wurst.hack.antiknockback": "让你不会被其他生物推动或击退。",
+  "description.wurst.hack.antispam": "预防刷屏，用计数器显示重复内容。",
+  "description.wurst.hack.antiwaterpush": "防止你被水流推动。",
+  "description.wurst.hack.antiwobble": "关闭因反胃和传送门引起的摇晃效果。",
+  "description.wurst.hack.autoarmor": "自动管理你的盔甲。",
+  "description.wurst.hack.autobuild": "自动建筑。\n放置一个方块以开始建造。",
+  "description.wurst.hack.autodrop": "自动丢弃你不想要的物品。",
+  "description.wurst.hack.autoleave": "当生命值过低时，自动退出服务器。",
+  "description.wurst.hack.autoeat": "让你自动进食。",
+  "description.wurst.hack.autofarm": "自动收获或种植作物。\n有效作物：小麦、胡萝卜、马铃薯、甜菜根、南瓜、西瓜、仙人掌、甘蔗、海带、竹子、地狱疣、可可豆。",
+  "description.wurst.hack.autofish": "自动使用最好的鱼竿钓鱼。",

```

Comment:
The another sentence is repeat of the first one.

Reply:
>  "description.wurst.hack.autofish": "Automatically catches fish using your best fishing rod. If it finds a better rod while fishing, it will automatically switch to it.",
> --[original English version](https://github.com/Wurst-Imperium/Wurst7/blob/master/src/main/resources/assets/wurst/lang/en_us.json#L16)

These are two different sentences.

Reply:
I think it is better
自动使用最好的鱼竿进行钓鱼，钓鱼过程中如果有更好的钓竿将自动切换。